src/EditorFeatures/Core/FindUsages/IFindUsagesService.cs

```diff
@@ -24,4 +24,37 @@ internal interface IFindUsagesService : ILanguageService
         /// </summary>
         Task FindImplementationsAsync(Document document, int position, IFindUsagesContext context);
     }
+
+    internal interface IFindUsagesServiceRenameOnceTypeScriptMovesToExternalAccess : ILanguageService
```

Comment:
I feel like a comment here could be useful for clarity (my first instinct was to ask "rename to what?"), or maybe just a link back to this PR/issue?

Reply:
Gotcha. This is fairly standard with moving something to EA. The point is to make it clear in the name that this is temporary. Once ts moves, we just rename this to the original simple name.